### ISSUE TITLE
Parametric EQ version with channel select per band

### ISSUE THREAD

**Issue by trummerschlunk** [sadko4u/lsp-plugins#241]
Dear Vladimir and associates ;)

I am working in professional audio mastering for almost 20 years, slowly transitioning from macOS to linux - it's a bumpy road, but I am almost there.

That said, your parametric EQ is the EQ-plugin of choice for me. It can do (almost) anything I need. Btw: the new GUI is really beautiful (and David packaged it already for arch <3)

A probably not so hard to implement addition would significantly speed up my workflow though. Could you imagine, making a version of parametric EQ 16 where every band gets one more dropdown menu that selects the target channel(s) of the band: stereo, mid, side, left, right ?

That would be awesome! I know, it's a difficult situation right now and I hope you and your family are ok. So no rush. Hopefully see you around soon!

Cheers, Klaus

**Comment by sadko4u** [comment]
Hello!

This requires a bit different look at the equalizer.
In this case, the equalizer should always be stereo (even if it processes mono signal) or contain two inputs and two outputs.
This is not so hard to implement the new equalizer but that requires some time.

As a point of curiosity: how does this help to your workflow? I mean, it would be nice to understand your user cases of the feature you're asking to implement.

**Comment by trummerschlunk** [comment]
Hey :)

Yes, the plugin version would always have 2 channels of input and output.

In my (mastering) workflow, every band of the EQ has a certain function. For instance one band cuts frequencies below 200Hz in the side signal to make the low-end mono (also known from elliptical filters in vinyl cutting). Another band might aim at pushing certain frequencies in the mid signal around 400Hz in order to make a vocalist sound warmer. Another classic is to push high frequencies in the sides with a shelf at maybe 10k to ‘widen’ the high-end. Some more bands I typically use in stereo mode to cut certain resonances that might be in the mix.

In conclusion, it’s very comfortable to have all this happen in one EQ instance and not having to jump between different instances.

Does this make sense to you?
Thank you so much for considering <3

Klaus

**Comment by sadko4u** [comment]
The main problem is, what name to introduce for the new equalizer :). Probably `Mastering Equalizer` or something similar.

**Comment by trummerschlunk** [comment]
Cool :)

Or 'unified eq'
or 'universal eq'
or 'uni-eq'

?

**Comment by trummerschlunk** [comment]
I am so curious... Are you working on this version? <3

**Comment by sadko4u** [comment]
No, at this moment I'm fixing bugs that I got while doing the huge update from 1.1.x to 1.2.0.
Probably I'll start working on this within few months.

**Comment by trummerschlunk** [comment]
Cool, thank you :)

**Comment by Daniele71** [comment]
Yeah, this would be very cool (and usefull).
